### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by ZapeeoSheikh** [community/community#71685] [GitHub Education]
### Hi everyone,

Hi 👋
This is Muhammad Rameez, I am a final year computer science student at Government college university lahore. I have served as a Google Developer Student Club Lead at my campus. Currently, I am a Beta Microsoft Learn Student Ambassador. I love flutter. I am a mobile app development enthusiast. I have build numerous problem solving mobile apps. 

Linkedin: https://www.linkedin.com/in/muhammadrmz/
Portfolio: https://mrameezportfolio.z10.web.core.windows.net/
Github: https://github.com/ZapeeoSheikh

**Comment by drdixit** [comment]
👋Hello @ZapeeoSheikh
